www/src/data/sidebars/doc-links.yaml

```diff
@@ -188,26 +188,26 @@
         - title: Styling Your Site
           link: /docs/styling/
           items:
-            - title: Using Layout Components
-              link: /docs/layout-components/
-            - title: Using CSS Modules
+            - title: Standard styling with global CSS files
+              link: /docs/global-css/
+            - title: Locally-scoped styles with CSS Modules
               link: /docs/css-modules/
-            - title: Using Typography.js
-              link: /docs/typography-js/
-            - title: Using CSS-in-JS Library Emotion
-              link: /docs/emotion/
-            - title: Using CSS-in-JS Library Glamor
-              link: /docs/glamor/
-            - title: Using CSS-in-JS Library Styled Components
-              link: /docs/styled-components/
-            - title: Creating Global Styles
-              link: /docs/creating-global-styles/
-            - title: Writing Component CSS
-              link: /docs/component-css/
-            - title: Using PostCSS
-              link: /docs/post-css/
-            - title: Using TailwindCSS
-              link: /docs/tailwind-css/
+            - title: Enhance your styles with CSS-in-JS
+              link: /docs/css-in-js/
+              items:
+                - title: Emotion
+                  link: /docs/emotion/
+                - title: Styled-Components
+                  link: /docs/styled-components/
+            - title: CSS Libraries and Frameworks
+              link: /docs/css-libraries-and-frameworks/
+              items:
+                - title: Typography.js
```

Comment:
Now that I'm learning more, we will want to move this to a new section on `Working with Typography and Fonts`. But it's okay to do it outside of the scope of this PR as it's a slightly different workflow. I'll be filing an issue for that in the next few days.

Reply:
Sounds great! I think it'll be better over there :slightly_smiling_face: